Jenkinsfile

```
@@ -447,7 +447,7 @@ stage('Non-default environments') {
 				withMavenWorkspace {
 					// Re-run integration tests against the JARs produced by the default build,
 					// but using a different JDK to build and run the tests.
-					mavenNonDefaultBuild buildEnv, "-f integrationtest"
+					mavenNonDefaultBuild buildEnv, "-pl " + sh(script: "./ci/list-all-integration-tests.sh", returnStdout: true).trim() + ":hibernate-search-util-common"
```

Or just this and remove the copy-pasted script:

```suggestion
					mavenNonDefaultBuild buildEnv, "-pl " + sh(script: "./ci/list-dependent-integration-tests.sh hibernate-search-engine", returnStdout: true).trim() + " -pl :hibernate-search-util-common"
```